common-travel.lic

```diff
@@ -81,15 +81,30 @@ module DRCT
     end
   end
 
-  def find_empty_room(search_rooms, idle_room, predicate = nil)
+  def find_empty_room(search_rooms, idle_room, predicate = nil, min_mana = false, strict_mana = false)
+
+    check_mana = min_mana
```

Comment:
Same as above. I was trying to limit the number of settings and arguments. I can just change the logic to be if > 0.